# Automatically Generated in 2017-06-12 22:33:47.453020
# MESSAGE: PARKING_LOT
# Description:Parking Lot Sensor
uint8 ID = 150
uint8 sysid
uint8 compid
uint32 timestamp
float32 parking_lot_size
float32 parking_lot_position
